js/portfolio.js

```diff
@@ -41,10 +31,26 @@ tabContent.forEach(function(a){
   $('#projects').append(a.toHtml())
 });
 
+schoolData.sort(function(a,b) {
+  return (new Date(b.publishedOn)) - (new Date(a.publishedOn));
+});
+
+schoolData.forEach(function(ele) {
+  tabContent.push(new Portfolio(ele));
+})
+
 tabContent.forEach(function(a){
```

Comment:
Here, tabContent is your projects, but you're appending it to the #school section. Is this intentional?